cfme/tests/configure/test_remote_server_tabs.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+@pytest.fixture(scope="module")
+def configured_external_appliance(temp_appliance_preconfig, app_creds_modscope,
+                                  temp_appliance_unconfig):
+    hostname = temp_appliance_preconfig.hostname
+    temp_appliance_unconfig.appliance_console_cli.configure_appliance_external_join(hostname,
+        app_creds_modscope['username'], app_creds_modscope['password'], 'vmdb_production',
+        hostname, app_creds_modscope['sshlogin'], app_creds_modscope['sshpass'])
+    temp_appliance_unconfig.evmserverd.start()
+    temp_appliance_unconfig.evmserverd.wait_for_running()
+    temp_appliance_unconfig.wait_for_web_ui()
+    return temp_appliance_unconfig
+
+
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.10')
```

Comment:
Required:
If this test not applicable to `5.10` then please use `ignore_stream`. We use `uncollectif`, when we like to `uncollect` for specific `attribute`, `object` or some `condition`. 

you can say some condition except only `stream`. I would like to use `only` in my statement because we will get some condition where we checked both `stream` and other `condition`.